src/main/java/org/opensrp/search/BaseSearchBean.java

```diff
@@ -0,0 +1,35 @@
+package org.opensrp.search;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class BaseSearchBean {
+
+	public enum OrderByType {
+		ASC, DESC
+	}
+
+	public enum FieldName {
+		id
+	}
+
+	private Integer pageNumber = 0;
+
+	private Integer pageSize = 0;
+
+	private BaseSearchBean.OrderByType orderByType;
+
+	private BaseSearchBean.FieldName orderByFieldName;
+
+	public BaseSearchBean(Integer pageNumber, Integer pageSize, OrderByType orderByType,
```

Comment:
This is not needed just add an `@AllArgsConstructor`

Reply:
Done